### ISSUE TITLE
Review Design System Team Current Members Write / Admin Access?

### ISSUE THREAD

**Issue by BeniCheni** [pricelinelabs/design-system#303]
Do you think it'd be helpful to review the current "Write" v.s. "Admin" access of the **pricelinelabs/design-system** team members?

I think it'd be helpful to list have an understanding of the current access of the design system project. I proposed a process in #302 that:

- 2 minimal approvals from **pricelinelabs/design-system** team members with "_Write_" access.

- A more "_restricted_" level of access would publish design system releases with "_Admin_" access

If that process is liked and updated to implement to the design system project, I think having the review of the current design system members' access would be help. I can also help out with this activity if anyone agrees. (P.S.: Using "_question_" & "_discussion_" labels to track this issue)

**Comment by jxnblk** [comment]
I wouldn't want to create different tiers of access to the design system. Currently everyone in the org should have *owner* privileges, and although the approval process is sort of casual right now, I think the Friday meetings and Slack channel should be the place to make approval decisions for the time being. We should probably clarify that a little more in the contributing doc.

If there's a particular problem we're trying to solve with this, I'd be happy to continue the discussion further

**Comment by BeniCheni** [comment]
Yeah, fair point and I agree that we want to keep the current Design System workflow "_light_" & "_streamlined_", as well as keeping the access privileges simple until the DS scales to a phase, where it would need more "_fine-tuning_" of the process.

Really enjoyed the workflow of our last Friday meeting to review PRs & issues, as well as the V2 discussion. So I also concur that Friday meetings and Slack channel are viable communication channel for the project now.

> We should probably clarify that a little more in the contributing doc.

We can use another open issue to narrow the discussion topic about contributing doc. In fact, I just got notifications about comments from #302. I raised this issue thinking that the org permission audit would work with #302. But having a quick look at the comment in #302, the discussion there seems to be evolving around the topic of how to update to mature the contributing doc.

So we can discuss more in #302, and I can close this issue.
